parm/config/gfs/config.resources

```diff
@@ -442,6 +442,35 @@ case ${step} in
     export memory_ocnanalecen
     ;;
 
+  "ocnanalletkf")
+    npes=16
+    case ${OCNRES} in
+      "025")
+        npes=40
```

Comment:
might as well be realistic, start with npes=480 for the 1/4 deg case

Reply:
done